README.md

```diff
@@ -1 +1,45 @@
-# awesome-code-connector
\ No newline at end of file
+<div align="center">
+<a href="https://codeconnector.io/" target="_blank">
+	<img src="assets/ccLogos/codeconnectorLogo.png" alt="codeconnector">
+</a>
+	<br>
+	<h4>Open Source Learning Resources, compiled by the Dev Community for the Dev Community.</h4>
```

Comment:
I don't think this repo should be limited to open source learning resources. It can be anything that code connector folks agree to aggregate.

Reply:
Okay will remove Open Source Learning Resources and replace with a more generalized term "Resources".